src/share/vm/jfr/dcmd/jfrDcmds.hpp

```diff
@@ -166,6 +166,25 @@ class JfrConfigureFlightRecorderDCmd : public DCmdWithParser {
   virtual void execute(DCmdSource source, TRAPS);
 };
 
+class JfrUnlockCommercialFeaturesDCmd : public DCmd {
+public:
+  JfrUnlockCommercialFeaturesDCmd(outputStream* output, bool heap) : DCmd(output, heap) { }
+  static const char* name() { return "VM.unlock_commercial_features"; }
+  static const char* description() {
+    return "Simulate commercial features unlocking for Zulu.";
```

Comment:
Maybe we should refine it as "Simulate commercial features unlocking for Alibaba Dragonwell8."